src/sentry/incidents/subscription_processor.py

```diff
@@ -562,6 +622,104 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         # before the next one then we might alert twice.
         self.update_alert_rule_stats()
 
+    def get_anomaly_detected(
+        self, anomaly
+    ) -> bool:  # TODO: add the anomaly type once that's added to Sentry
+        """
+        Helper function to determine whether we care about an anomaly based on the
+        anomaly type and the alert rule's threshold type.
+        """
+        # need to define constants for anomaly types
+        anomaly_type = anomaly.get("anomaly").get("anomaly_type")
+
+        if anomaly_type == "none" or anomaly_type == "no_data":
+            return False
+        elif anomaly_type == "anomaly_high":
+            if (
+                self.alert_rule.threshold_type == AlertRuleThresholdType.ABOVE.value
+                or self.alert_rule.threshold_type == AlertRuleThresholdType.ABOVE_AND_BELOW.value
+            ):
+                return True
+        else:
+            if (
+                self.alert_rule.threshold_type == AlertRuleThresholdType.BELOW.value
+                or self.alert_rule.threshold_type == AlertRuleThresholdType.ABOVE_AND_BELOW.value
+            ):
+                return True
+        return False
+
+    def get_anomaly_data_from_seer(self, aggregation_value: float | None):
+        try:
+            ad_config = {
+                "time_period": self.alert_rule.threshold_period,
+                "sensitivity": self.alert_rule.sensitivity,
+                "seasonality": self.alert_rule.seasonality,
+                "direction": self.alert_rule.detection_type,
+            }
+
+            context = {
+                "id": self.alert_rule.id,
+                "cur_window": {
+                    "timestamp": self.last_update,
+                    "value": aggregation_value,
+                },
+            }
+            response = make_signed_seer_api_request(
+                self.seer_anomaly_detection_connection_pool,
+                SEER_ANOMALY_DETECTION_URL,
+                json.dumps(
```

Comment:
This should be the URL for the `detect_anomalies_endpoint`, e.g. https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L3417-L3429 so for this it would be the URL from the decorator here: https://github.com/getsentry/seer/blob/f0b1cfd9cf9c04300307b549bd36c38fc1097c8a/src/seer/app.py#L248